.travis.yml

```diff
@@ -19,9 +19,10 @@ jobs:
   include:
     - env: TEST="sbt ci-test"
       script: sbt ci-test
-    - env: TEST="sbt ci-test"
-      scala: 2.11.12
-      script: sbt ci-test
+    # Disabled until mtags dependency of server is published for 2.11
```

Comment:
I'm working on a PR for metals that I want to complete before cross-build for 2.11